functioning logout button with logout toast added

zsofinnemeth opened the pull request:
(This may not really belong to this branch but we did not want to start a new branch for this small feature)

zsofinnemeth committed:
functioning logout button with logout toast added

adamgyulavari commented on app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:
```diff
@@ -27,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
         Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);
         setSupportActionBar(myToolbar);
         textViewWelcome = (TextView) findViewById(R.id.textViewWelcome);
+        logoutButton =(Button) findViewById(R.id.logoutButton);
```
@zsofinnemeth pls move this into the options menu

zsofinnemeth replied:
moved to the options menu of the toolbar

zsofinnemeth replied:
@adamgyulavari

zsofinnemeth committed:
cropped redundant textview ("Welcome back")

zsofinnemeth committed:
Merge branch 'zsuzsi-actionbar' of https://github.com/greenfox-zerda-raptors/ZerdaReaderAndroid into zsofi-appbar

zsofinnemeth committed:
Merge branch 'master' of https://github.com/greenfox-zerda-raptors/ZerdaReaderAndroid into zsofi-appbar

# Conflicts:
#	app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java
#	app/src/main/res/layout/activity_main.xml

zsofinnemeth committed:
switch bug fixed (added breaks)

adamgyulavari commented on app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:
```diff
@@ -11,13 +11,16 @@
 
 public class MainActivity extends AppCompatActivity {
 
+    Button logoutButton;
```
@zsofinnemeth you don't use this variable

adamgyulavari commented on app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:
```diff
@@ -49,6 +52,17 @@ private void checkIfLoggedIn() {
     }
 
+    public void logOut(){
+       SharedPreferences loginData = PreferenceManager.getDefaultSharedPreferences(this);
```
@zsofinnemeth you already have a sharedpreferences reference, you don't need another.

zsofinnemeth replied:
sorry badly handled merge conflict.. in a minute

adamgyulavari commented on app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:
```diff
@@ -49,6 +52,17 @@ private void checkIfLoggedIn() {
+        editor.apply();
+        Toast.makeText(this,"Successful logout", Toast.LENGTH_SHORT).show();
+        Intent loginIntent = new Intent(MainActivity.this, LoginActivity.class);
+        startActivity(loginIntent);
```
@zsofinnemeth pay attention to indentation!

zsofinnemeth replied:
@adamgyulavari  I hope I got it right by now!

zsofinnemeth committed:
code duplication fixed

zsofinnemeth committed:
indentation fixed

adamgyulavari commented on app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java:
```diff
@@ -42,13 +42,25 @@ protected void onCreate(Bundle savedInstanceState) {
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+            editor.putString("userName", "");
+            editor.putString("password", "");
+            editor.apply();
```
@zsofinnemeth almost :) these 3 lines are not in a new block, so they should be indented as the same as the above editor creation. Also resolve conflicts and then this PR is mergable.

zsofinnemeth replied:
@adamgyulavari  fixed! (?)

zsofinnemeth committed:
conflicts fixed Merge branch 'master' of https://github.com/greenfox-zerda-raptors/ZerdaReaderAndroid into zsofi-appbar

# Conflicts:
#	app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java

zsofinnemeth committed:
code duplication after conflict handling fixed
